src/locale/locales/ca/messages.po

```diff
@@ -458,7 +458,7 @@ msgstr "Afegeix els canals recomanats"
 
 #: src/screens/StarterPack/Wizard/index.tsx:488
 msgid "Add some feeds to your starter pack!"
-msgstr ""
+msgstr "Afegiu persones que creus que els agradaria seguir als altres al teu starter pack"
```

Comment:
This looks like it's for a different string?

Reply:
Thank you very much for all your corrections!